### PR TITLE
refactor: rename `lyne-components` into `lyne-elements` and create `lyne-elements-experimental`

### LINKED ISSUE #2617: Split packages into lyne-elements, lyne-elements-experimental, lyne-react and lyne-react-experimental
We have the following directory structure and corresponding package names:

- src/elements (`@sbb-esta/lyne-elements`)
- src/elements-experimental (`@sbb-esta/lyne-elements-experimental`, has a dependency to `@sbb-esta/lyne-elements`)
- src/react (`@sbb-esta/lyne-react`, has a dependency to `@sbb-esta/lyne-elements`)
- src/react-experimental (`@sbb-esta/lyne-react-experimental`, has a dependency to `@sbb-esta/lyne-elements-experimental` and `@sbb-esta/lyne-elements`)

Components to move:

- journey-summary
- pearl-chain (all)
- timetable-duration
- timetable-row
- Maybe also move core/timetable

Storybook should have separate sections for each package with the package name as label.

### PR DESCRIPTION
**Pull request by TomMenga**
## Brief description
This PR Closes #2617 

We now have the following libraries:
- `@sbb-esta/lyne-elements` => `src/elements`
- `@sbb-esta/lyne-elements-experimental` => `src/elements-experimental` (has a dependency to `@sbb-esta/lyne-elements`)
- `@sbb-esta/lyne-react` => `src/react` (has a dependency to `@sbb-esta/lyne-elements`)
- `@sbb-esta/lyne-react-experimental` =>  `src/react-experimental` (has a dependency to `@sbb-esta/lyne-elements-experimental` and `@sbb-esta/lyne-elements`)

### Topics to discuss:
- I marked the inter-dependencies as peer dependency. That forces the user to explicitly have them in their package.json. I feel like it's better this way (also cdk does it). What do you think?
- ~~Adapt the CI/CD (probably just the release-please)~~

-----

BREAKING CHANGE: `Lyne components` and `Lyne components react` libraries have been renamed, please update imports accordingly.
- `@sbb-esta/lyne-components` => `@sbb-esta/lyne-elements`
- `@sbb-esta/lyne-components-react` => `@sbb-esta/lyne-react`

The following components, and their react wrappers, have been moved into `@sbb-esta/lyne-elements-experimental` and `@sbb-esta/lyne-react-experimental` respectively:
- sbb-journey-summary
- sbb-pearl-chain
- sbb-pearl-chain-time
- sbb-pearl-chain-vertical
- sbb-pearl-chain-vertical-item
- sbb-timetable-duration
- sbb-timetable-row

### CODE REVIEW

**Review by kyubisation — COMMENTED** [review]
Excellent work! 😃

**Comment by kyubisation** [comment]
Additional thought; Should we add an explanation for the experimental packages to the root README? And maybe a quick additional section to the GETTING_STARTED.md?

**Comment by kyubisation** [comment]
Suggestion: "The experimental packages provide components that have either not yet been battle tested or do not yet align with our architecture or testing goals. Once they become stable, we will move them into the main packages."

File: src/react/experimental-vite.config.ts
[the remark this answers is not in the document]
Comment (TomMenga) [reply src/react/experimental-vite.config.ts:1]: Makes sense, you convinced me 👍

**Comment by TomMenga** [comment]
> Additional thought; Should we add an explanation for the experimental packages to the root README? And maybe a quick additional section to the GETTING_STARTED.md?

Docs updated 😸 
I took the liberty of slightly changing the suggestion, see if you like it (took inspiration from angular/components) @kyubisation  @jeripeierSBB

**Review by kyubisation — APPROVED** [review]

**Review by kyubisation — APPROVED** [review]
LGTM :+1
Thank you very much!

**Pull request merged**